mycroft/util/format.py

```diff
@@ -34,7 +34,6 @@
 from lingua_franca.format import (NUMBER_TUPLE, DateTimeFormat, join_list,
                                   date_time_format, expand_options,
                                   _translate_word)
```

Comment:
importing a private object is a no-no.  If we are importing it, make it public.

Reply:
Agree.

This was not introduced by this PR. A quick look tells me that 
- this private method is being used for `nice_duration()` and `nice_duration_dt()`. 
- currently `lingua_franca.format.nice_duration()` is not directly compatible with `mycroft.util.format.nice_duration()`
- `nice_duration_dt` (duration between two datetimes) doesn't exist in LF - but presumably we can just get the timedelta and call `nice_duration()`, or add it to LF.
- Making `nice_duration()` compatible will require some work outside the scope of this PR. We possibly want to look at what the available arguments are too. There are a few bools to format it in different ways but I'm not clear on what happens if a dev passes True for conflicting arguments eg `nice_duration(duration, speech=True, clock=True)`
   - (I think speech wins and clock gets ignored)

I've added #TODO's at least. Doing more of the above will be much more refactoring and may require breaking changes to core and/or Lingua Franca.

Reply:
`nice_duration_dt` was my mediocre doing, and I'd still have to go back and look at it =P it needs to be migrated or die, probably. Should happen when `nice_duration` is reconciled.

Reply:
Have also added an issue in mycroft-core that links to the issue about this in LF.